airbyte-integrations/connectors/source-zoho-crm/metadata.yaml

```diff
@@ -2,44 +2,46 @@ data:
   ab_internal:
     ql: 200
     sl: 100
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:2.0.0@sha256:c44839ba84406116e8ba68722a0f30e8f6e7056c726f447681bb9e9ece8bd916
```

Comment:
@pabloescoder can you bump this to 3.0 while we're here? I want to send this over to progressive rollout and see if we can ship this.